test/csuite/wt8246_compact_rts_data_correctness/main.c

```diff
@@ -255,6 +255,8 @@ workload_compact(const char *home, const char *table_config, const char *uri)
 
     testutil_check(session->checkpoint(session, NULL));
 
+    testutil_check(session->checkpoint(session, "force"));
```

Comment:
Why do we need to force a checkpoint immediately after doing a checkpoint? If this is needed please add a comment explaining why we want the forced checkpoint.

Reply:
Added a comment, thanks